lib/services/elastic-helpers.js

```diff
@@ -182,8 +188,9 @@ function convertRedisBatchtoElasticBatch(index, type, ops) {
         indexOp._id = op.key;
       }
 
-      bulkOps.push({ index: indexOp },
-        op.value
+      bulkOps.push(
+        update ? { update: indexOp } : { index: indexOp },
```

Comment:
`index` and `update` are just a couple of options that can be passed in, could this get adjusted to accept either `index`, `update`, `delete` or `create`, like the Elastic docs illustrate? https://www.elastic.co/guide/en/elasticsearch/reference/2.4/docs-bulk.html